tests/menus/menu_items_registry.test.ts

```diff
@@ -1315,6 +1315,22 @@ describe("Menu Item actions", () => {
     });
   });
 
+  test("Format -> indentation", () => {
+    doAction(["format", "format_increase_indent"], env);
+    expect(dispatch).toHaveBeenCalledWith("SET_FORMATTING", {
```

Comment:
Lucas-sensei once told be that it was better to check the effect of the menu item in the sheet, rather than checking that a command was dispatched 🤓